client/src/featureform/register.py

```diff
@@ -1522,7 +1522,7 @@ def register_snowflake(
         snowflake = ff.register_snowflake(
             name="snowflake-quickstart",
             username="snowflake",
-            password="password",
+            password="password", #pragma: allowlist secret
```

Comment:
unsure what this will look like on the docs, but we'll see. Having issues running the docs locally